src/styles/_About.scss

```diff
@@ -0,0 +1,31 @@
+@use 'variables' as v;
+
+.about {
+  font-family: v.$secondary-font;
+  min-height: 100vh;
+  background-color: v.$secondary-background-color;
+  .about__header {
+    display: flex;
+    align-items: center;
+    font-size: 1.5rem;
+    background-image: url('../images/about-bg.jpg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center center;
```

Comment:
When I looked at the landing page in fullscreen, the image is focused on the center which looks odd for this image. When I tried `background-position: top` it looked alright in all the different screen sizes. 
What do you think?